packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -89,6 +89,21 @@ storiesOf("Worldview/GLText", module)
       </Container>
     );
   })
+  .add("glyph ascent and descent", () => {
+    const markers = textMarkers({ text: "BDFGHJKLPQTY\nbdfghjklpqty", billboard: true });
+    const target = markers[9].pose.position;
+    return (
+      <Container
+        cameraState={{
```

Comment:
nit: `cameraState` usually require `onCameraStateChange` together
```suggestion
        defaultCameraState={{
```

Reply:
@vidaaudrey Please correct me if I'm wrong, but since I'm using the `<Container />` class I don't need to specify `onCameraStateChange`, right?